tools/Ensembl-REST/.shed.yml

```diff
@@ -1,6 +1,6 @@
 ---
 name: Ensembl-REST
-owner: tgac
+owner: Earlham Institute
```

Comment:
Mmmh, `tgac` is the ToolShed user "public name", which cannot be changed unfortunately. Should I create a new ToolShed user? `tgac` has 2 published repositories (hcluster_sg and miranda), we can deprecate them and publish them again under the new user.